docs/io/fits/appendix/faq.rst

```diff
@@ -52,19 +52,15 @@ bases is non-trivial.  The second is that there are many features of Astropy
 greatly.  Since PyFITS is already integrated into Astropy, it makes more sense
 to continue development there rather than make Astropy a dependency of PyFITS.
 
-PyFITS' past primary developer and active maintainer was Erik Bray.  There
```

Comment:
I think we can keep this in there for history, maybe just remove the word "active". (after all we keep all the mentions of people from the very in the contributing page)

Reply:
Okay. Done. Thanks!